scripts/build-mocks.sh

```diff
@@ -9,6 +9,9 @@ cd $GOPATH/src/github.com/mesg-foundation/core
 # generate mocks for container package.
 mockery -name=Container -dir ./container -output ./container/mocks
 
+# generate mocks for docker.CommonAPIClient that used by container package.
+mockery -name=Docker -dir ./utils/docker -inpkg
```

Comment:
out file - mock_Docker.go -> don't use capital letters in filename.

Reply:
I couldn't see a configuration related to this in mockery's cli. You can open an issue on their repo page. But I think it's OK this way.

Reply:
`-case underscore`

Reply:
I tried it and applied it to all other mocks but as I see, it messes with the interface names as well. For example, _ServiceDB.go_ mock file for _ServiceDB_ interface becomes to _service_db.go_ and I don't like this naming. I prefer to skip setting the case, it's not that important anyway.